### ISSUE TITLE
explanation of the split jsons

### ISSUE THREAD

**Issue by Tgaaly** [ondyari/FaceForensics#18]
I see the split jsons here: https://github.com/ondyari/FaceForensics/tree/master/dataset/splits
Is there any explanation for this. Why are they a list of pairs of numbers and what do these numbers reference? There should be 720 train videos, 140 validation and testing videos. I see that all the numbers correspond to those numbers correctly but I'm just wondering why they are listed in pairs?

**Comment by vinthony** [comment]
same question

**Comment by vinthony** [comment]
I found something. The pairs mean the `source` and `target` corresponding in the manipulated sequences.

For example, the pair with '210_219' folder means the background video is from 210 while the face is from 219.

**Comment by Tgaaly** [comment]
yes I thought of that. this means that it is also works the other way round, since there are 140 numbers in the test split so for each pair, the first number would be the source for the second and vice versa. 

Indeed I found these correspond to videos in the data. For example the first pair of test.json is 953, 974. If you look under DeepFakes directory you will find two videos 953_974.mp4 and 974_953.mp4. The same goes for the other subsets.

**Comment by nkuo** [comment]
what about original videos? (for negative instances for deepfake detection) what is the split for that?

**Comment by CesiumLifeJacket** [comment]
Re: negative instances, I suspect it works like this: If train.json contains ["123", "456"], this means the training set includes the original videos 123 and 456, and the manipulated videos 123_456 and 456_123. This is just a guess, though. It would be nice to see this spelled out explicitly.

**Comment by hamzag95** [comment]
I imported the json as a data frame via pandas. It displayed as a two column data frame. However, which column is real or fake? 

Also even the fake videos have a number identifier 0-999, so you can just use the first three is what I was thinking. At first I thought the same thing as @Tgaaly though

**Comment by ucalyptus2** [comment]
@ondyari can u explain how to load these json and make them dataloaders?

**Comment by xi4444x** [comment]
> @ondyari can u explain how to load these json and make them dataloaders?
I have the same question ,do you already know how to get it ?
